config/default/translations/messages-en.properties

```diff
@@ -131,4 +131,428 @@ task.pnc.danger_sign_followup_mother.title = PNC danger sign follow up - mother
+report.pregnancy_facility_visit_reminder.data.meta=
+report.pregnancy_facility_visit_reminder.data.__patient_uuid=Patient UUID
+report.pregnancy_facility_visit_reminder.data.__patient_id=Patient ID
+report.pregnancy_facility_visit_reminder.data.__household_uuid=Household uuid
+report.pregnancy_facility_visit_reminder.data.__source=Source
+report.pregnancy_facility_visit_reminder.data.__source_id=Source ID 
+report.pregnancy_facility_visit_reminder.data.__pregnancy_uuid=Pregnancy UUID
+report.pnc_danger_sign_follow_up_baby.danger_signs = Danger Sign Follow-up - Baby
+report.pnc_danger_sign_follow_up_baby.danger_signs.visit_confirm = Baby taken to the health facility as recommended
+report.pnc_danger_sign_follow_up_baby.danger_signs.danger_sign_present = Baby still experiencing any danger signs
+report.pnc_danger_sign_follow_up_baby.danger_signs.danger_signs_question_note = Danger signs.
+report.pnc_danger_sign_follow_up_baby.danger_signs.infected_umbilical_cord = Infected umbilical cord
+report.pnc_danger_sign_follow_up_baby.danger_signs.convulsion = Convulsions
+report.pnc_danger_sign_follow_up_baby.danger_signs.difficulty_feeding = Difficulty feeding or drinking
+report.pnc_danger_sign_follow_up_baby.danger_signs.vomit = Vomits everything
+report.pnc_danger_sign_follow_up_baby.danger_signs.drowsy = Drowsy or unconscious
+report.pnc_danger_sign_follow_up_baby.danger_signs.stiff = Body stiffness
+report.pnc_danger_sign_follow_up_baby.danger_signs.yellow_skin = Yellow skin color
+report.pnc_danger_sign_follow_up_baby.danger_signs.fever = Fever
+report.pnc_danger_sign_follow_up_baby.danger_signs.blue_skin = Blue skin color (hypothermia)
+report.pnc_danger_sign_follow_up_baby.danger_signs.refer_patient_note_1 =  Please refer to the health facility immediately. Accompany her if possible.
+report.pnc_danger_sign_follow_up_baby.danger_signs.refer_patient_note_2 =  Please complete the follow-up task within 3 days.
+report.pnc_danger_sign_follow_up_mother.danger_signs = Danger Sign Follow-up - Mother
+report.pnc_danger_sign_follow_up_mother.danger_signs.visit_confirm = Patient visited the health facility as recommended
+report.pnc_danger_sign_follow_up_mother.danger_signs.danger_sign_present =Still experiencing any danger signs
+report.pnc_danger_sign_follow_up_mother.danger_signs.danger_signs_question_note = Danger signs
+report.pnc_danger_sign_follow_up_mother.danger_signs.fever = Fever
+report.pnc_danger_sign_follow_up_mother.danger_signs.severe_headache = Severe headache
+report.pnc_danger_sign_follow_up_mother.danger_signs.vaginal_bleeding = Vaginal bleeding
+report.pnc_danger_sign_follow_up_mother.danger_signs.vaginal_discharge = Foul smelling vaginal discharge
+report.pnc_danger_sign_follow_up_mother.danger_signs.convulsion = Convulsions 
+report.pnc_danger_sign_follow_up_mother.danger_signs.refer_patient_note_2 =  Please advise her to do so and accompany her if possible. 
```

Comment:
```suggestion
report.pnc_danger_sign_follow_up_mother.danger_signs.refer_patient_note_2 =  Please advise her to do so and accompany her if possible
```

Reply:
Also, this label is unclear as to what the instruction is... "to do so" could be "to visit a health facility immediately"?

Reply:
```suggestion
report.pnc_danger_sign_follow_up_mother.danger_signs.refer_patient_note_2 =  Please advise her to visit the health facility immediately and accompany her if possible 
```